pkg/splunk/enterprise/indexercluster_test.go

```diff
@@ -95,7 +96,58 @@ func TestApplyIndexerCluster(t *testing.T) {
 	splunkDeletionTester(t, revised, deleteFunc)
 }
 
-func getIndexerClusterPodManagerAndClient(method string, mockHandlers []spltest.MockHTTPHandler, replicas int32) (*indexerClusterPodManager, *spltest.MockHTTPClient) {
+func TestGetClusterMasterClient(t *testing.T) {
+	scopedLog := log.WithName("TestGetClusterMasterClient")
+	cr := enterprisev1.IndexerCluster{
+		TypeMeta: metav1.TypeMeta{
+			Kind: "IndexerCluster",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "stack1",
+			Namespace: "test",
+		},
+		Spec: enterprisev1.IndexerClusterSpec{
+			Replicas: 1,
+			CommonSplunkSpec: enterprisev1.CommonSplunkSpec{
+				ClusterMasterRef: corev1.ObjectReference{
+					Name: "", /* Empty ClusterMasterRef */
+				},
+			},
+		},
+		Status: enterprisev1.IndexerClusterStatus{
+			ClusterMasterPhase: splcommon.PhaseReady,
+		},
+	}
+	cr.Status.IndexerSecretChanged = append(cr.Status.IndexerSecretChanged, true)
```

Comment:
Do we need this?

Reply:
Hmm yeah, good catch, I don't think so we need this. Typical copy-paste issue :)